app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -432,6 +432,13 @@ define([
             },
             $.mage.__('Please enter a valid phone number. For example (123) 456-7890 or 123-456-7890.')
         ],
+        'validate-phone': [
+            function (value) {
+                return utils.isEmptyNoTrim(value) ||
+                    /^[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$/.test(value);
+            },
+            $.mage.__('Please enter a valid phone number. For example (123) 456-7890, 123-456-7890, +(123)4567890, +1234567890')
```

Comment:
Static tests are failing because of this line length, can you please break it into multiple lines?

Reply:
Fixed by add `//eslint-disable-line max-len` on the end string
 like on the others cases in this file
Thanks for advice